modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1154 @@
+
+=== Added common/space/delBetweenExclamationMarks to remove spaces between exclamation marks
+// TINY-9398		
+
+An issue was discovered that affected the typography plugin in previous versions of {productname} when inserting content into the editor. Specifically, the issue involved exclamation marks and the spaces between them, such as in the example `“Hello! !“`. This issue prevented the editor from removing the spaces between the double *exclamation marks* `! !` after executing the typography plugin toolbar or menubar button.
+
+The fix for this involved adding a new rule which removes the spaces between exclamation marks when the typography button is pressed.
+
+In {productname} 6.4, spaces are now removed between exclamation marks `!!` for formatted HTML documents when the typography plugin is used. To learn more about Advanced Typography and how to add this new premium plugin to your project, please refer xref:advanced-typography.adoc[Advanced Typography].
+
+=== Added typography plugin to the default menubar
+// TINY-9450		
+
+In previous versions of {productname}, the typography button would *not* appear in the `menubar` by default. As a consequence, the user could only `find` the typography button in the `toolbar` due to it not appearing in the `menubar` dropdown.
+
+To correct this bug, the typography plugin was added to the default `menubar` config.
+
+{productname} 6.4, by default, will now display the typography button in both the `toolbar` and below the “Source code" button in the `menubar`.
+
+=== Typography button and menu item will be disabled if editor has no content
+// TINY-9441
+
+In previous versions of {productname}, the typography plugin button would render to the toolbar when the editor contained no content.
+
+As a consequence of this, users would be able to apply the typography button features to an editor that contained no physical content.
+
+As a fix, {productname} added some new improvements to the Advanced Typography plugin that now disables the users ability to apply the plugin and use its features, when the editor content value returns as `empty`.
+
+In {productname} 6.4, applying the typography button to an empty editor, will no longer be possible.
+
+For information on the **Advanced Typography** plugin, see: xref:advanced-typography.adoc[Advanced Typography].
+
+////
+Be wary of the version number listed above.
+
+To quote my [comment, left on the TinyMCE 6.4 BoM][0] in Notion:
+
+> Given the five changes in the Unreleased section of the Advanced
+> Typography plugin’s [changelog][1], is it sensible to assume this plugin
+> is getting a version bump?
+
+[0]: <https://notion.so/TinyMCE-6-4-Bill-of-materials-BOM-885a34e3b9884be0b69cd30eba7c618b?d=5267c02d507140c2882a18c2a9c7c753#0067a51f89bc45f1a99cc1b80a6e7a39>
+
+[1]: <https://github.com/tinymce/typography/blob/main/CHANGELOG.md>
+////
```

Comment:
```suggestion
```
There's nothing in the unreleased section any more, the current release (in 6.4.1) is version 1.1.0 of advanced typography, as is visible in the changelog.
I can see multiple comments in this file regarding "Unreleased" items in premium plugins, though they are all likely released with the correct new version numbers by now.